src/modules/user-settings/lib/editor.ts

```diff
@@ -14,29 +15,50 @@ export const supportedEditorConfig: Record< SupportedEditor, SupportedEditorConf
 		label: __( 'VS Code' ),
 		url: ( path: string ) => `vscode://file/${ path }?windowId=_blank`,
 		bundleId: 'com.microsoft.VSCode',
+		winPaths: [
+			'%LOCALAPPDATA%\\Programs\\Microsoft VS Code\\code.exe',
+			'%PROGRAMFILES%\\Microsoft VS Code\\code.exe',
+			'%PROGRAMFILES(X86)%\\Microsoft VS Code\\code.exe',
+		],
 	},
 	phpstorm: {
 		// translators: "PhpStorm" is the brand name for an IDE and does not need to be translated
 		label: __( 'PhpStorm' ),
 		url: ( path: string ) => `phpstorm://open?file=${ path }`,
 		bundleId: 'com.jetbrains.PhpStorm',
+		winPaths: [
+			'%LOCALAPPDATA%\\Programs\\PhpStorm\\bin\\phpstorm64.exe',
+			'%PROGRAMFILES%\\JetBrains\\PhpStorm\\bin\\phpstorm64.exe',
```

Comment:
I am afraid that these absolute paths are not guaranteed. I just installed PhpStorm on my Windows, and it was installed in `JetBrains\PhpStorm 2025.1.0.1\bin\phpstorm64.exe`. 

JetBrains products use `<product><version>` pattern for directories. You can read more about it at: https://www.jetbrains.com/help/phpstorm/directories-used-by-the-ide-to-store-settings-caches-plugins-and-logs.html#config-directory

Reply:
Thanks for sharing, I also saw that docs page earlier. Just a small clarification, this link refers to the directories Jetbrains uses to store config, caches, and logs, not the installation paths.

Could you share if `JetBrains\PhpStorm 2025.1.0.1\bin\phpstorm64.exe` is within Program Files or your user App Data folder for you?
 
Based on what I find on the internet, I believe the path doesn't contain a version number for IDEs installed for the current user only (this is the default in the Toolbox app). I could also verify this on my VMs.

For IDEs installed for all users, it should go to Program Files, but I'm unsure about the exact folder structure here.

Reply:
Using the [standalone installer](https://www.jetbrains.com/help/webstorm/installation-guide.html#standalone) I could verify that for system-wide installs, the path should be `%PROGRAMFILES\JetBrains\<product> <version>`.

![CleanShot 2025-05-08 at 11 50 32@2x](https://github.com/user-attachments/assets/c8c9b63b-5469-4710-8a6b-eccc45c40eab)

Reply:
I updated the matching logic in 7ca0c5a4e4a62b531c599c87f278aaad8681c357